### PR TITLE
Chapter 8: base-login

### PR DESCRIPTION
**Pull request by vannam1996**

### CODE REVIEW

**Comment by vannam1996** [comment]
image

**Comment by vannam1996** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: dùng current_user.present? nhé

**Comment by dothidiemthao** [comment]
commented

**Comment by vannam1996** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**